trestle/tasks/cis_xlsx_to_oscal_catalog.py

```diff
@@ -281,19 +281,27 @@ def _create_property(self, name: str, value: str) -> Property:
 
     def _add_property(self, xlsx_helper: XlsxHelper, props: List[Property], row: int, key: str) -> None:
         """Add property."""
-        name = self._get_normalized_name(key)
-        value = xlsx_helper.get(row, key)
-        if value:
-            props.append(self._create_property(name, value))
+        try:
```

Comment:
Are there required columns with information used to create properties? Wondering if this would resulting in unexpected errors because the change is not specific to `status` and the `mitre` columns.

Reply:
> Are there required columns with information used to create properties? Wondering if this would resulting in unexpected errors because the change is not specific to `status` and the `mitre` columns.

Thx for the comment. None of the properties are required.

Reply:
Note to reviewers: this fix is for a specific issue encountered when working with the CIS folks for the CIS Talos Linux Benchmark. There may be further generalizations and improvements as other CIS Benchmarks are explored, for example sheet names other than the 2 hard-coded in the present transformer. Not gonna try to fix everything in this PR.